modules/ROOT/pages/secure-storage.adoc

```diff
@@ -0,0 +1,110 @@
+= Secure Storage Overview
+
+Anypoint Studio 7.10 and earlier versions stored the information of authenticated users in the workspace, meaning that you could have different authenticated users per workspace.
+
+Studio 7.11 introduces a Secure Storage facility to encrypt and save your credentials.
+
+Studio provides a master password provider that automatically generates a master password for all your user credentials. When adding a new credential, Studio uses your master password to encrypt and save that credential into a secure storage file in your file system.
+
+Because of the architecture of the secure storage, the authenticated users are now shared across all Eclipse-based workspaces and Anypoint Studio instances.
+
+== Master Password Providers
+
+The available master password providers depend on your operative system. Studio assigns a priority to the available master password providers and enables you to choose which one to use. By default, Studio assigns a higher level of priority to the password provider that uses your OS's underlying capabilities to generate and encrypt your master password.
```

Comment:
Maybe I would be less assertive that it is Studio who do things. It is really Eclipse, we just use the default providers and behavior defined by Eclipse.